tensorboard/webapp/hparams/_redux/hparams_actions.ts

```diff
@@ -50,3 +54,11 @@ export const hparamsMetricFilterChanged = createAction(
     includeUndefined: boolean;
   }>()
 );
+
+export const hparamsFetchSessionGroupsSucceeded = createAction(
```

Comment:
Do you have plans to introduce the other actions we typically also have for data source fetches? 

`hparamsFetchSessionGroupsRequested`
`hparamsFetchSessionGroupsFailed`

(I'd ok - in fact, I prefer - if these come in a different PR. I just want to make sure they are on your radar).

Reply:
Yeah, I'll make sure these get added soon. I wanted to get the MVP out as soon as possible to unblock James.